corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowser.java

```diff
@@ -209,9 +221,10 @@ public int dropTable(String namespace, String tablename) {
      * @param tablename - table name without the namespace
      * @param numItems - total number of items to load
      * @param batchSize - number of items in each transaction
+     * @param itemSize - size of each item - a random string array
      * @return - number of entries in the table
      */
-    public int loadTable(String namespace, String tablename, int numItems, int batchSize) {
+    public int loadTable(String namespace, String tablename, int numItems, int batchSize, int itemSize) {
```

Comment:
I did not realize earlier, but in this operation we allow data to be loaded only in the schema of dummyKey and dummyVal, but the clients table can have a different schema, right?  Wont the load operation fail?

Reply:
this is not meant to load data into client tables - if they do that they are intentionally corrupting data. This tool is just to load specific data into corfu.

Reply:
I see, ok.  what is the use case of this operation?  is it for testing disk full like you mentioned in another comment?  Should we add it in the comments section for this method?  

Also, if accidentally, this is called for a client table, the operation will fail with incompatible schema, right?  If not, is it better to control from Corfu itself which table the data gets loaded into?  It can be a fixed table inside CorfuSystem or something.

Reply:
its for disk full and load testing. added comments to state the same.

Reply:
dint find the comments...